K Framework LLVM Backend Build Error

I get the following error when trying to build K from source

```
[INFO] ---------------< com.runtimeverification.k:llvm-backend >---------------
[INFO] Building K Framework LLVM Backend 1.0-SNAPSHOT                    [9/10]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-java) @ llvm-backend ---
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ llvm-backend ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO]
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ llvm-backend ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- maven-antrun-plugin:1.7:run (build-llvm) @ llvm-backend ---
[INFO] Executing tasks

main:
     [exec] -- Found LLVM 6.0.0
     [exec] -- Found GMP: /usr/lib/x86_64-linux-gnu/libgmp.so
     [exec] -- Boost version: 1.65.1
     [exec] -- Found the following Boost libraries:
     [exec] --   unit_test_framework
     [exec] -- Configuring done
     [exec] -- Generating done
     [exec] -- Build files have been written to: /home/joshpoll/k/llvm-backend/target/build
     [exec] [  4%] Built target AST
     [exec] [  4%] Generating datastructures/target/release/libdatastructures.rlib
     [exec] [ 17%] Built target Hello
     [exec] [ 20%] No such file or directory
     [exec] make[2]: *** [runtime/datastructures/target/release/libdatastructures.rlib] Error 1
     [exec] make[1]: *** [runtime/CMakeFiles/datastructures.dir/all] Error 2
     [exec] make[1]: *** Waiting for unfinished jobs....
     [exec] Built target strings
     [exec] [ 28%] Built target Parser
     [exec] [ 32%] Built target arithmetic
     [exec] [ 35%] Built target meta
     [exec] [ 39%] Built target io
     [exec] [ 45%] Built target alloc
     [exec] Scanning dependencies of target runtime-arithmetic-tests
     [exec] Scanning dependencies of target hello
     [exec] Scanning dependencies of target runtime-io-tests
     [exec] Scanning dependencies of target parser
     [exec] Scanning dependencies of target configurationparser
     [exec] Scanning dependencies of target runtime-strings-tests
     [exec] runtime/CMakeFiles/datastructures.dir/build.make:65: recipe for target 'runtime/datastructures/target/release/libdatastructures.rlib' failed
     [exec] CMakeFiles/Makefile2:753: recipe for target 'runtime/CMakeFiles/datastructures.dir/all' failed
     [exec] [ 46%] Building CXX object unittests/runtime-arithmetic/CMakeFiles/runtime-arithmetic-tests.dir/floattest.cpp.o
     [exec] [ 48%] Building CXX object tools/parser/CMakeFiles/parser.dir/main.cpp.o
     [exec] [ 50%] Building CXX object unittests/runtime-io/CMakeFiles/runtime-io-tests.dir/io.cpp.o
     [exec] [ 51%] Building CXX object unittests/runtime-arithmetic/CMakeFiles/runtime-arithmetic-tests.dir/inttest.cpp.o
     [exec] [ 53%] Building CXX object tools/hello/CMakeFiles/hello.dir/main.cpp.o
     [exec] [ 56%] Building CXX object unittests/runtime-arithmetic/CMakeFiles/runtime-arithmetic-tests.dir/main.cpp.o
     [exec] [ 56%] Building CXX object unittests/runtime-strings/CMakeFiles/runtime-strings-tests.dir/stringtest.cpp.o
     [exec] [ 57%] Building CXX object runtime/configurationparser/CMakeFiles/configurationparser.dir/ConfigurationParser.cpp.o
     [exec] [ 59%] Building CXX object unittests/runtime-io/CMakeFiles/runtime-io-tests.dir/main.cpp.o
     [exec] [ 60%] Building CXX object runtime/configurationparser/CMakeFiles/configurationparser.dir/ConfigurationPrinter.cpp.o
     [exec] Scanning dependencies of target Codegen
     [exec] [ 62%] Building CXX object unittests/runtime-strings/CMakeFiles/runtime-strings-tests.dir/bytestest.cpp.o
     [exec] [ 64%] Building CXX object lib/codegen/CMakeFiles/Codegen.dir/CreateTerm.cpp.o
     [exec] [ 65%] Linking CXX executable hello
     [exec] [ 65%] Built target hello
     [exec] [ 67%] Building CXX object lib/codegen/CMakeFiles/Codegen.dir/Decision.cpp.o
     [exec] [ 68%] Building CXX object lib/codegen/CMakeFiles/Codegen.dir/DecisionParser.cpp.o
     [exec] [ 70%] Linking CXX executable parser
     [exec] [ 71%] Linking CXX static library libconfigurationparser.a
     [exec] [ 71%] Built target configurationparser
     [exec] [ 73%] Building CXX object lib/codegen/CMakeFiles/Codegen.dir/EmitConfigParser.cpp.o
     [exec] [ 75%] Building CXX object lib/codegen/CMakeFiles/Codegen.dir/GenAlloc.cpp.o
     [exec] [ 76%] Building CXX object unittests/runtime-strings/CMakeFiles/runtime-strings-tests.dir/main.cpp.o
     [exec] [ 76%] Built target parser
     [exec] [ 78%] Linking CXX executable runtime-io-tests
     [exec] [ 79%] Linking CXX executable runtime-strings-tests
     [exec] [ 79%] Built target runtime-io-tests
     [exec] [ 81%] Linking CXX executable runtime-arithmetic-tests
     [exec] [ 82%] Linking CXX static library libCodegen.a
     [exec] [ 82%] Built target Codegen
     [exec] [ 82%] Built target runtime-strings-tests
     [exec] make: *** [all] Error 2
     [exec] [ 82%] Built target runtime-arithmetic-tests
     [exec] Makefile:129: recipe for target 'all' failed
```

I'm running Ubuntu 18.04. I ran the following commands:
```
git submodule update --init --recursive
sudo apt-get install build-essential m4 openjdk-8-jdk libgmp-dev libmpfr-dev pkg-config flex z3 libz3-dev maven opam python3 cmake gcc clang-6.0 clang++-6.0 llvm-6.0 llvm-6.0-tools zlib1g-dev bison libboost-test-dev libyaml-cpp-dev libjemalloc-dev
llvm-backend/src/main/native/llvm-backend/install-rust
curl -sSL https://get.haskellstack.org/ | sh
```
and I modified my `~/.bashrc` with the following:
```
export PATH=/home/joshpoll/.local/bin:$PATH
export MAVEN_OPTS=-XX:+TieredCompilation
export PATH=/home/k/k-distribution/target/release/k/bin:$PATH
```

The error above was generated from running `mvn package -e` in `~/k`. I also tried running `mvn clean` and then `mvn package -e` again, but it produced the same error.

Any help towards getting K to build is appreciated. Thanks!

--- a/llvm-backend/pom.xml
+++ b/llvm-backend/pom.xml
@@ -55,11 +55,14 @@
                   <arg value="-DCMAKE_BUILD_TYPE=${project.build.type}" />
                   <arg value="${project.basedir}/src/main/native/llvm-backend" />
                 </exec>
+                <property environment="env"/>
                 <exec executable="make" dir="${project.build.directory}/build" failonerror="true">
                   <arg value="-j12" />
+		  <env key="PATH" value="${env.PATH}:${env.HOME}/.cargo/bin" />
                 </exec>
                 <exec executable="make" dir="${project.build.directory}/build" failonerror="true">
                   <arg value="install" />
+		  <env key="PATH" value="${env.PATH}:${env.HOME}/.cargo/bin" />
                 </exec>
               </target>
             </configuration>

